### DISCUSSION TITLE
Segmentation Fault

### DISCUSSION THREAD

**Discussion by cody-cao** [openai/whisper#1284] [Q&A]
Hi all, 

Quick question: I keep running into this segmentation error when trying to run whisper. I installed everything the way the code page mentioned yet to no avail :/

(base) xxxx@mac-xx Desktop % whisper audio.wav
zsh: segmentation fault  whisper audio.wav

**Comment by lyaotian** [comment]
I don't know what system and what GPU you are using, may be this links can help you.
https://stackoverflow.com/questions/76104114/segfault-when-using-pytorch-with-rocm-in-linux
https://bugs.archlinux.org/task/78306
